Remove no longer needed testing infrastructure from post-commit OpenSource

robert-jenner opened the pull request:
#### d0941d28f74c55fcfbcd60f512740b321d5611c7
<pre>
Remove no longer needed testing infrastructure from post-commit OpenSource
<a href="https://rdar.apple.com/127372525">rdar://127372525</a>
<a href="https://bugs.webkit.org/show_bug.cgi?id=273562">https://bugs.webkit.org/show_bug.cgi?id=273562</a>

Reviewed by Ryan Haddad.

This fix removes no longer needed testing infrastructure from the configuration. It also adds Ventura JSC testers.

* Tools/CISupport/build-webkit-org/config.json:
* Tools/CISupport/build-webkit-org/factories_unittest.py:
(TestExpectedBuildSteps):
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/Dashboard.js:
* Tools/CISupport/build-webkit-org/public_html/dashboard/Styles/Main.css:
(table.queue-grid tr.platform.macos-monterey img.logo): Deleted.
* Tools/CISupport/build-webkit-org/steps_unittest.py:

Canonical link: <a href="https://commits.webkit.org/278238@main">https://commits.webkit.org/278238@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/60450d09c0d932a20b746e1e18432bdf7e479ba3

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/49951 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/29239 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/2233 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/53194 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/628 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/52252 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/35304 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/197 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/40747 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/52049 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/26808 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/42972 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/21876 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/24239 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/207 "Passed tests") | [✅ 🛠 wpe-skia](https://ews-build.webkit.org/#/builders/52/builds/8321 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/46335 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/255 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/54775 "Built successfully") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/44/builds/25045 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/62/builds/186 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/48137 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/49973 "Passed tests") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/26303 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/43170 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/47186 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/27165 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/7201 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/26032 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

ryanhaddad commented on Tools/CISupport/build-webkit-org/config.json:
```diff
@@ -33,22 +33,22 @@
+                  { "name": "bot198", "platform": "*" },
+                  { "name": "bot279", "platform": "*" },
+                  { "name": "bot603", "platform": "*" },
+                  { "name": "bot632", "platform": "*" },
```
This should be platform: mac-ventura and be moved to the section with the other Ventura bots.

ryanhaddad commented on Tools/CISupport/build-webkit-org/config.json:
```diff
@@ -33,22 +33,22 @@
-                  { "name": "bot1025", "platform": "mac-monterey" },
-                  { "name": "bot1026", "platform": "mac-monterey" },
+                  { "name": "bot105", "platform": "*" },
+                  { "name": "bot111", "platform": "*" },
```
This should be platform: mac-ventura and be moved to the section with the other Ventura bots.

ryanhaddad commented on Tools/CISupport/build-webkit-org/config.json:
```diff
@@ -33,22 +33,22 @@
-                  { "name": "bot1023", "platform": "mac-monterey" },
-                  { "name": "bot1025", "platform": "mac-monterey" },
-                  { "name": "bot1026", "platform": "mac-monterey" },
+                  { "name": "bot105", "platform": "*" },
```
This should be platform: mac-ventura and be moved to the section with the other Ventura bots.

ryanhaddad commented on Tools/CISupport/build-webkit-org/config.json:
```diff
@@ -33,22 +33,22 @@
+                  { "name": "bot187", "platform": "*" },
+                  { "name": "bot198", "platform": "*" },
+                  { "name": "bot279", "platform": "*" },
+                  { "name": "bot603", "platform": "*" },
```
This should be platform: mac-ventura and be moved to the section with the other Ventura bots.

ryanhaddad reviewed (changes requested):
Left a few comments, also need to make changes to `Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/WebKitBuildbot.js` for the queues that are moving.

robert-jenner committed:
Remove no longer needed testing infrastructure from post-commit OpenSource
rdar://127372525
https://bugs.webkit.org/show_bug.cgi?id=273562

Reviewed by Ryan Haddad.

This fix removes no longer needed testing infrastructure from the configuration. It also adds Ventura JSC testers.

* Tools/CISupport/build-webkit-org/config.json:
* Tools/CISupport/build-webkit-org/factories_unittest.py:
(TestExpectedBuildSteps):
* Tools/CISupport/build-webkit-org/public_html/dashboard/Scripts/Dashboard.js:
* Tools/CISupport/build-webkit-org/public_html/dashboard/Styles/Main.css:
(table.queue-grid tr.platform.macos-monterey img.logo): Deleted.
* Tools/CISupport/build-webkit-org/steps_unittest.py:

Canonical link: https://commits.webkit.org/278238@main
